test/unit/server/rpc/procedures/baidu-maps.spec.js

```diff
@@ -0,0 +1,96 @@
+describe('baidu-staticmaps', function() {
```

Comment:
if we want the two services to have the same API maybe we can have them share the tests.